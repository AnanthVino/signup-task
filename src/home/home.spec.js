import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Home from '.';
    
describe('Home', () => {
  it('should render correctly', () => {
    
    const wrapper = shallow(<Home />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})