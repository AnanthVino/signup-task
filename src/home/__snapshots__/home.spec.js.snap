// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
<div
  className="App"
>
  <nav
    className="navbar navbar-expand-lg navbar-dark text-light bg-primary"
  >
    <h3>
      Ajar Online
    </h3>
  </nav>
  <div
    className="d-flex justify-content-center"
  >
    <div
      className="d-flex home-box"
    >
      <div
        className="form-header text-light bg-primary text-center"
      >
        <h5>
          Sign Up
        </h5>
      </div>
      <div
        className="d-flex flex-row justify-content-center align-items-center flex-grow-1"
      >
        <Link
          className="btn btn-info rounded-0"
          to="/landlord"
        >
          As Landlord
        </Link>
        <Link
          className="btn btn-info rounded-0"
          to="/admin"
        >
          As Admin
        </Link>
      </div>
    </div>
  </div>
</div>
`;
